build.gradle

```diff
@@ -13,11 +13,12 @@ configurations {
 }
 
 dependencies {
-    compile 'io.crate:crate:1.0.2'
+    compile 'io.crate:crate:2.0.2'
+    compile 'org.apache.logging.log4j:log4j-api:2.7'
     dependency 'net.rtme:classnamerj:0.0.1'
```

Comment:
as you replaced the nice classnamer scalar with a boring is_even scalar ;( this dependency is no longer needed.